<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new hotkey</string>
  </property>
  <widget class="QLineEdit" name="hotkey_line_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hotkey_label_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotkey:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sound_file_label_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sound file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sound_file_line_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save_hotkey">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_sound_file_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_note">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: Depending on your system many more formats than just wav or mp3 could be supported. If you wish to pick any other format select &quot;All files&quot; when file select dialog opens. Be sure to test other formats, if they don't play they are not supported.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear_hotkey_entry">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_listen_hotkey">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_copy_sound_file">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy sound file to sounds directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_help_copy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>111</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
